units/commit_queue.go

```diff
@@ -104,165 +104,175 @@ func (j *commitQueueJob) Run(ctx context.Context) {
 		return
 	}
 	nextItem := cq.Next()
-	if nextItem == "" {
-		return
-	}
-
-	nextItemInt, err := strconv.Atoi(nextItem)
-	if err != nil {
-		j.AddError(errors.Wrapf(err, "can't parse next item \"%s\" as int", nextItem))
-		grip.Error(message.WrapError(err, message.Fields{
-			"job":     commitQueueJobName,
-			"source":  "commit queue",
-			"project": j.QueueID,
-			"item":    nextItem,
-			"message": "next item can't be parsed as an int",
-		}))
-
-		_, err = cq.Remove(nextItem)
-		j.AddError(errors.Wrapf(err, "error dequeuing item '%s'", nextItem))
-		grip.Error(message.WrapError(err, message.Fields{
-			"job":     commitQueueJobName,
-			"source":  "commit queue",
-			"project": j.QueueID,
-			"item":    nextItem,
-			"message": "can't dequeue item",
-		}))
-
+	if nextItem == nil {
```

Comment:
A lot of refactoring and moving things around in this file.
Most of the real changes were just to get the module PRs and informations about them to append to the patch (lines 143-154) and to subscribe the module PRs to be merged (lines 176-181)